src/cogent3/app/composable.py

```diff
@@ -129,651 +129,6 @@ def to_json(self):
-                    continue
-
-                logger.log_message(outcome, label="output")
-                logger.log_message(outcome.md5, label="output md5sum")
-
-        if am_writer:
-            finish = time.time()
-            taken = finish - start
-
-            logger.log_message(f"{taken}", label="TIME TAKEN")
-            logger.shutdown()
-            self.data_store.add_file(log_file_path, cleanup=cleanup, keep_suffix=True)
-            self.data_store.close()
-
-        # now reconnect input
-        if process is not self:
-            self = process + self
-
-        return results
-
-
-class ComposableTabular(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableTabular, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableTabular",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class ComposableSeq(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableSeq, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableSeq",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class ComposableAligned(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableAligned, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableAligned",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class ComposableTree(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableTree, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableTree",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class ComposableModel(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableModel, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableModel",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class ComposableHypothesis(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableHypothesis, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableHypothesis",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class ComposableDistance(Composable):  # pragma: no cover
-    def __init__(self, **kwargs):
-        super(ComposableDistance, self).__init__(**kwargs)
-        discontinued(
-            "class",
-            "ComposableDistance",
-            "2023.1",
-            "see developer docs for new class hierarchy",
-        )
-
-
-class _checkpointable:
```

Comment:
I am not sure about how we replaced this class in new decorator.

Reply:
All support for checkpointing is now in `app_type=WRITER`.